playground/frontend/lib/modules/examples/example_selector.dart

```diff
@@ -159,12 +160,16 @@ class _ExampleSelectorState extends State<ExampleSelector>
                             width: kLgContainerWidth,
                             decoration: BoxDecoration(
                               color: Theme.of(context).backgroundColor,
-                              borderRadius: BorderRadius.circular(kMdBorderRadius),
+                              borderRadius:
+                                  BorderRadius.circular(kMdBorderRadius),
                             ),
                             child: exampleState.sdkCategories == null ||
```

Comment:
This 7-line ternary is hard to read.
It can be placed into `_buildDropdownContent`. More so because `LoadingIndicator` is still the dropdown content.